CHANGELOG.md

```diff
@@ -66,6 +66,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * [#1573](https://github.com/NibiruChain/nibiru/pull/1573) - feat(perp): Close markets and compute settlement price
 * [#1705](https://github.com/NibiruChain/nibiru/pull/1705) - feat(perp): Add oracle pair to market object
 * [#1718](https://github.com/NibiruChain/nibiru/pull/1718) - fix: fees does not require additional funds
+* [#1734](https://github.com/NibiruChain/nibiru/pull/1734) - feat(perp): MsgDonateToPerpFund
```

Comment:
A little bit of a more descriptive message like the one in the title would be nice. You will appreciate it when you search for this in the changelog in four years!